lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,6 +1,6 @@
 # These dependencies are only required when certain config options are set
 psycopg2==2.6.1
-WebError==0.11
+WebError==0.10.3
```

Comment:
I pushed the tweak removing it from the core requirements.txt and it appears to fetch and work correctly, and conditionally.